core/datastore-proto/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+plugins {
+    id("nowinandroid.android.library")
+    alias(libs.plugins.protobuf)
+}
+
+android {
+    namespace = "com.google.samples.apps.nowinandroid.core.datastore"
+}
+
+// Setup protobuf configuration, generating lite Java and Kotlin classes
+protobuf {
```

Comment:
If this is a workaround, can we document it? Maybe add some comments in this file?